doc/configuration/privacyidea-appliance.rst

```diff
@@ -112,14 +112,56 @@ Restore
 The entry `View Backups` will list all the backups available.
 
 .. figure:: images/appliance/backup3.png
-   :scale: 50%
+   :width: 550px
 
    All available backups
 
 You can select a backup and you are asked if you want to restore the data. 
 
 .. warning:: Existing data is overwritten and will be lost.
 
+.. _database:
+
+Database: Setup Redundancy
+..........................
+
+The appliance-tool is also capable of setting up a redundant setup, between
+two privacyIDEA nodes. For the setup, the privacyIDEA slave instance has to
+be setup and running with identical software than the master instance. The
+appliance-tool uses a tinc-tunnel to share the configuration and the database.
+
+.. figure:: images/appliance/database.png
+   :width: 400px
+
+.. note:: The redundancy requires to setup a tinc VPN connection between
```

Comment:
I don't think the tinc tunnel is required, will check.

Reply:
tinc tunnel is not required.

Reply:
okay